app/templates/gentelella/users/events/event_base.html

```diff
@@ -107,16 +107,19 @@
                     <button type="button" class="close" data-dismiss="modal" aria-label="Close"><span
                             aria-hidden="true">&times;</span></button>
                     <h4 class="modal-title">{{ _("Are you sure you would like to delete this event") }} ?</h4>
+                    <span>{{ _("Deleting the event will delete all the data associated with it.") }} <br><br>{{ _("Please enter the full name of the event to continue") }}</span><br>
+                    <input id = 'delete-check' type = "text"/ data-eventname = "{{ event.name }}">
```

Comment:
Let's give proper names. For attributes separate multiple words with a hyphen. So `data-eventname` would become `data-event-name`